internal/utils/misc.go

```diff
@@ -148,7 +149,7 @@ var (
 	FallbackImportMapPath = filepath.Join(FunctionsDir, "import_map.json")
 	FallbackEnvFilePath   = filepath.Join(FunctionsDir, ".env")
 	DbTestsDir            = filepath.Join(SupabaseDirPath, "tests")
-	SeedDataPath          = filepath.Join(SupabaseDirPath, "seed.sql")
+	DefaultSeedDataPath   = filepath.Join(SupabaseDirPath, "seed.sql")
```

Comment:
Q: is it possible to remove this derived field altogether? so we can have `Db.Seed.SqlPaths` as the single source of truth

Reply:
I think we can, only issue is at init time, we create a seed file by writing the "seed.sql" file onto `supabase/` folder.

But maybe we don't need the "write a default" file logic if we just skip the seed if the default `seed.sql` isn't present.